src/main/kotlin/org/rust/cargo/toolchain/Rustup.kt

```diff
@@ -81,4 +87,43 @@ class Rustup(
         val fs = LocalFileSystem.getInstance()
         return fs.refreshAndFindFileByPath(FileUtil.join(sysroot, "lib/rustlib/src/rust/src"))
     }
+
+    companion object {
+
+        // We don't want to install Clippy if:
+        // 1. It is already installed
+        // 2. We don't have Rustup
+        // 3. Rustup doesn't have Clippy component
```

Comment:
Let's move this comment into `checkNeedInstallComponent` and generalize its text